tests/MongoDB.Driver.Tests/Linq/Linq2ImplementationTestsOnLinq3/Translators/AggregateGroupTranslatorTests.cs

```diff
@@ -429,10 +429,12 @@ public void Should_translate_aggregate_expressions_with_user_provided_serializer
         private ProjectedResult<TResult> Group<TKey, TResult>(Expression<Func<Root, TKey>> idProjector, Expression<Func<IGrouping<TKey, Root>, TResult>> groupProjector, ExpressionTranslationOptions translationOptions)
         {
             var queryable = __collection.AsQueryable3()
-                .GroupBy(idProjector)
-                .Select(groupProjector);
+                                        .GroupBy(idProjector)
+                                        .Select(groupProjector);
```

Comment:
We always indent 4 spaces.

Horizontal real estate is too "valuable" to waste like this.

And trying to maintain vertical alignment is  not worth it (everytime you rename something or change something you have to move things left and right to keep them aligned).

Reply:
My personal preference is to indent method chains so that the operations line up vertically, but I'm OK with following our coding conventions and doing a standard 4-space indent.